net/eoip/Makefile

```diff
@@ -0,0 +1,57 @@
+#
+# Copyright (C) 2006-2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
```

Comment:
Omit it or add your own copyright.

See https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md#makefile-contents-should-contain